### PR TITLE
[app-config] Update descriptions of the properties

### PR DESCRIPTION
**Pull request by minhanh-phan**
- Update the descriptions of the properties in the app-configuration Swagger file

### CODE REVIEW

**Review by jsquire — APPROVED** [review]
LGTM; please hold off on merging until we have at least one approver from the AppConfig service team.

File: specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json
Comment (jimmyca15) [inline specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json:183]: Do you mind to write it like this?

`A filter used get key-values for a snapshot. The value should be the name of the of the snapshot.`

Comment (minhanh-phan) [reply specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json]: That sounds good. Made the change accordingly!

**Review by jimmyca15 — APPROVED** [review]

**Review by HarshaNalluru — APPROVED** [review]

**Pull request merged**
